main.go

```diff
@@ -293,13 +293,13 @@ func configFromEnv(vargs *GAE, workspace *string) error {
 func validateVargs(vargs *GAE) error {
 
 	if vargs.Token == "" {
-		return fmt.Errorf("missing required param: token")
+		return fmt.Errorf("missing required credentials: GAE_CREDENTIALS or param token")
```

Comment:
Is the plugin looking for a `TOKEN` or `PLUGIN_TOKEN`?

Reply:
No, this plugin is looking for `GAE_CREDENTIALS`. I know, it's not the same. Maybe future improvement can look for multiple but the way it is written now it doesn't.

Reply:
I was referring more to the error message where it says `GAE_CREDENTIALS or param token`